modules/ROOT/pages/configure-teams.adoc

```diff
@@ -0,0 +1,126 @@
+= Configure Teams
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Every root organization has a main _parent team_ that appears by default. Every user in your organization is a member of the highest parent team and receives all permissions therein.
```

Comment:
Suggestion: We've been calling the auto-provisioned team the `root team` which pairs nicely with `root organization`

```suggestion
Every root organization has a _root team_ that appears by default. Every user in your organization is automatically a member of this root team and receives all permissions therein.
```